src/test/java/com/wikia/webdriver/common/core/imageutilities/ImageGenerator.java

```diff
@@ -3,12 +3,10 @@
 import com.wikia.webdriver.common.contentpatterns.PageContent;
 import com.wikia.webdriver.common.core.CommonUtils;
 import com.wikia.webdriver.common.logging.PageObjectLogging;
+
 import org.apache.commons.lang.exception.ExceptionUtils;
 
-import java.awt.Color;
-import java.awt.Font;
-import java.awt.FontMetrics;
-import java.awt.Graphics2D;
+import java.awt.*;
```

Comment:
true, i've used 'optimalize imports' option in Intellij
i found that wildcard option can be disabled